src/styles.ts

```diff
@@ -35,16 +35,17 @@ export const styles = css`
 			--cosmoz-input-focused-color,
 			var(--primary-color, #3f51b5)
 		);
+		--cosmoz-input-border-color: var(--line-color);
 		--contour-color: var(--line-color);
 		--contour-size: var(--cosmoz-input-contour-size);
+		--cosmoz-input-border-color: var(--contour-color);
+		--cosmoz-input-transform-y-percentage: 0%;
```

Comment:
The same comment as above. You should do:

```suggestion
		--transform-y-percentage: var(--cosmoz-input-transform-y-percentage, 0%);
```